### ISSUE TITLE
Republish sims that support Interactive Description on iOS 14

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#1082]
In https://github.com/phetsims/scenery/issues/1094 we addressed an issue with the PDOM to support accessibility on iOS 14, and it is working for RaP publication. But we never republished sims with the fix, so they still don't work with VoiceOver in the latest iOS. This includes all sims that we support with mobile interactive description:
- gravity-force-lab
- ~~gravity-force-lab-basics~~ EDIT: This was republished off of master this year and has been fixed.
- molarity
- molecules-and-light
- ohms-law

https://github.com/phetsims/scenery/issues/1094 required a lot of work, and created non-trivial side issues such as https://github.com/phetsims/scenery/issues/1137. I don't see a good way to do this in a maintenance release. Unfortunately, I think these will need to be fixed by republishing off of master. This would be a large chunk of work for the team so I wanted to discuss with @emily-phet if/when/how this should be done.

One way to do it is to prioritize sims that would not be published naturally. GFL will be republished Q2 and OL possibly in Q3. We could start by republishing molecules-and-light because it is the most fresh, then molarity, then GFL. But I know Q2 already feels pretty tight. @emily-phet feel free to put thoughts in this issue but it might be best to briefly discuss this in person.

**Comment by jessegreenberg** [comment]
If we delay or stagger republishing these, do we need to remove the "Interactive Description on Mobile Devices" flag from the sims where this is broken?

**Comment by terracoda** [comment]
I agree with @jessegreenberg, we will review the list of affected sims and un-publish the mobile a11y feature from the sims that cannot be fixed soon, put them back on the prototypes page with that feature.

**Comment by jessegreenberg** [comment]
This came up during planning for this quarter, and again during the quarter check-in meeting on 2/7/21 - we decided not to republish sims with this fix and that we would remove the "Interactive Description on Mobile Devices" flag on the website instead.

The sims that will not work with iOS 14 (but are marked as supporting the feature) are:
- gravity-force-lab
- molarity
- molecules-and-light
- ohms-law

Assigning to @terracoda to update the simulation metadata on the website for these sims.

**Comment by terracoda** [comment]
Thanks @jessegreenberg.

**Comment by terracoda** [comment]
@jessegreenberg, I think I might need dev versions of these 4 sims, so I have a link to publish on the prototypes page with the fix.

**Comment by terracoda** [comment]
Or perhaps we do not have fully working prototypes?

**Comment by jessegreenberg** [comment]
Discussed over slack with @terracoda - We haven't built dev versions of these sims since they were published to production so we don't have dev versions with this fix. We are going to build new versions off of master now for the prototypes page.

**Comment by jessegreenberg** [comment]
New dev versions for these sims have been deployed and handed off to @terracoda. Reassigning back to her.

**Comment by terracoda** [comment]
I have updated the website, unpublished the mobile A11y feature for the 4 remaining sims, published them to the sim prototypes page.
